src/useFormContext.tsx

```diff
@@ -7,7 +7,11 @@ const FormGlobalContext = React.createContext<FormContextValues<
 > | null>(null);
 
 export function useFormContext<T extends FieldValues>(): FormContextValues<T> {
-  return React.useContext(FormGlobalContext) as FormContextValues<T>;
+  const context = React.useContext(FormGlobalContext) as FormContextValues<T>;
+  if (context === undefined) {
+    throw new Error('useFormContext must be used within a FormContext');
+  }
+  return context;
```

Comment:
```suggestion
    return context || throw new Error('Missing FormContext');
```
Thanks a lot of the PR ❤️ 
I am shorten the message to reduce the bundle size :)